src/components/dialogs/NotInGoodStandingDialog.vue

```diff
@@ -13,16 +13,37 @@
           Business is not in good standing
         </p>
 
-        <p class="warning-text">
-          This business cannot be dissolved yet. Before you can dissolve your business it must be in good standing with
-          the Business Registry. There may be several reasons a business is not in good standing, but the most common
-          reason is an overdue annual report.
-        </p>
+        <template v-if="isDissolveMessage">
+          <p class="warning-text">
+            This business cannot be dissolved yet. Before you can dissolve your business, it must be
+            in good standing with BC Registries. There may be several reasons why a business is not
```

Comment:
@severinbeauvais leave this one as with "the Business Registry"

Reply:
done